src/UseCases/AddDonation/AddDonationUseCase.php

```diff
@@ -59,4 +73,68 @@ private function newBankDataFromRequest( AddDonationRequest $request ): BankData
 		return $bankData->freeze()->assertNoNullFields();
 	}
 
+	private function newDonationEntityFromDonationDomain( Donation $donation ) {
+		$dbalDonation = new \WMDE\Fundraising\Entities\Donation();
```

Comment:
This is misplaced. Fixed in #183

The idea is to not have anything that is a persistence implementation detail, such as the Store component or Doctine, outside of the DataAccess folder. (Well, also in the factory and tests possible, but certainly not in the Domain folder.) Not how the repository interfaces are in the Domain folder and the implementations are in DataAccess.